scripts/docker-compose/imports/keycloak/realm.json

```diff
@@ -944,6 +944,109 @@
         "offline_access",
         "microprofile-jwt"
       ]
+    },
+    {
+      "clientId": "zaakafhandelcomponent-admin-client",
+      "name": "zaakafhandelcomponent-admin-client",
+      "description": "",
+      "rootUrl": "",
+      "adminUrl": "",
+      "baseUrl": "",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "alwaysDisplayInConsole": false,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "zaakafhandelcomponentAdminClientSecret",
+      "redirectUris": [
+        "/*"
+      ],
+      "webOrigins": [
+        "/*"
+      ],
```

Comment:
can we make this stricter? or is this only used for running it locally in docker?

Reply:
Only for running locally in Docker. Also, it's just a spike.